torchtune/datasets/_text_completion.py

```diff
@@ -49,7 +49,7 @@ def __getitem__(self, index: int) -> Tuple[List[int], List[int]]:
         sample = self._data[index]
         return self._prepare_sample(sample)
 
-    def _prepare_sample(self, sample: Mapping[str, Any]) -> Tuple[List[int], List[int]]:
+    def _prepare_sample(self, sample: Mapping[str, Any]) -> Dict[str, List[int]]:
```

Comment:
you'll have to update the return type of the above function as well. running `pre-commit` should catch these